README.md

```diff
@@ -5,22 +5,26 @@ standard library, compiler, and language spec.
 
 # How to contribute
 
+Issues and bug reports for _scala/scala_ are located in [scala/bug](https://github.com/scala/bug). This tracker is also where new contributors may find good first issues to work on.
```

Comment:
I suggest making "good first issues" a hyperlink to https://github.com/scala/bug/labels/good%20first%20issue